sensor-vernier/src/main/java/org/concord/sensor/vernier/labpro/LabProSensorDevice.java

```diff
@@ -167,9 +172,18 @@ public final static int round(float value)
      * @see org.concord.sensor.device.SensorDevice#configure(org.concord.sensor.ExperimentRequest)
      */
     public ExperimentConfig configure(ExperimentRequest request) 
-    {
-    	return autoIdConfigure(request);
-    }
+	{
+		ExperimentConfig experimentConfig = autoIdConfigure(request);
+
+		// Configure the LabPro-specific timing delay
+		experimentConfig.setInitialReadDelay(INITIAL_READ_DELAY_MILLIS);
+
+		// Configure the LabPro library appropriately
+		int numChannels = experimentConfig.getSensorConfigs().length;
+		port.setNumChannels(numChannels);
```

Comment:
Originally I was hoping it would be enough to _only_ call it here, but ended up having to call it in start(). Given that start() always comes after configure(), I'll go ahead and remove this.